x-pack/solutions/security/test/security_solution_api_integration/test_suites/lists_and_exception_lists/exception_lists_items/trial_license_complete_tier/items/delete_exception_list_items.ts

```diff
@@ -65,6 +68,85 @@ export default ({ getService }: FtrProviderContext) => {
         );
       });
 
+      describe('@skipInServerless with read rules and all exceptions role', () => {
+        const role = ROLES.rules_read_exceptions_all;
+
+        beforeEach(async () => {
+          await createUserAndRole(getService, role);
+        });
+
+        afterEach(async () => {
+          await deleteUserAndRole(getService, role);
+        });
+        it('should delete a single exception list item by its item_id', async () => {
+          const restrictedUser = { username: 'rules_read_exceptions_all', password: 'changeme' };
+          const restrictedApis = exceptionsApi.withUser(restrictedUser);
+
+          // create an exception list
+          await supertest
+            .post(EXCEPTION_LIST_URL)
+            .set('kbn-xsrf', 'true')
+            .send(getCreateExceptionListMinimalSchemaMock())
+            .expect(200);
+
+          // create an exception list item
+          await supertest
+            .post(EXCEPTION_LIST_ITEM_URL)
+            .set('kbn-xsrf', 'true')
+            .send(getCreateExceptionListItemMinimalSchemaMock())
+            .expect(200);
+          const { item_id: itemId } = getCreateExceptionListItemMinimalSchemaMock();
+          // fixes type assertion
+          if (itemId == null) {
+            throw Error('broken test');
+          }
+          // delete the exception list item by its item_id
+          const { body } = await restrictedApis
+            .deleteExceptionListItem({ query: { item_id: itemId } })
+            .expect(200);
+
+          const bodyToCompare = removeExceptionListItemServerGeneratedProperties(body);
```

Comment:
Please use explicit assertions with `.toMatchObject()` or `expect.objectContaining()` instead of `removeExceptionListServerGeneratedProperties()`.

This comment is applicable to all FTR test files using `removeExceptionListServerGeneratedProperties`.